core/modules/contact/contact.module

```diff
@@ -5,6 +5,22 @@
  * Enables the use of personal and site-wide contact forms.
  */
 
+
+/**
+ * Implements hook_theme().
+ */
+function contact_theme() {
+  $base = array(
+    'file' => 'contact.theme.inc',
+  );
+
+  return array(
+    'contact_category_list' => array(
```

Comment:
Although theming the entire form works, it would be better to theme just the $form['categories'] element, so that other form elements may be placed around the form via hook_form_alter() if needed.